scss/style.scss

```diff
@@ -1,14 +1,8 @@
 // ==========================================================================
 // CSS Chassis
 // ==========================================================================
+// This just add normalize to the build see lint.scss for expaination.
 
 @import
-	"_utilities/clearfix",
-	"_utilities/colors";
-
-@import
-	"atoms/icons/icons",
-	"atoms/typography/typography";
-
-@import
-	"views/main";
+	"external/normalize.css/normalize.scss",
+	"lint.scss";
```

Comment:
If you look at the comments in the `lint.scss` and the comments and tasks for this in the grunt file it should be clear to contributors the purpose of this i think. Im open to a better way to do it but i don't think there is one unless we just don't lint our distributable which seems like a bad idea.